src/containers/Forms/Student/StudentForm.js

```diff
@@ -84,7 +95,14 @@ const StudentForm = ({ title, submitData, selectedStudent, id }) => {
           label='Язык обучения'
           type='text'
           required
-        />
+          select
+        >
+          {languages.map((option) => (
+            <MenuItem key={option.value} value={option.value}>
+              {option.value}
```

Comment:
Попробовал подставить, но все равно только value отображает
```suggestion
              {option.label}
```

Reply:
Исправлено